sdks/python/apache_beam/utils/annotations.py

```diff
@@ -90,6 +90,8 @@ def inner(*args, **kwargs):
       message = '%s is %s' % (fnc.__name__, label)
       if label == 'deprecated':
         message += ' since %s' % since
+        if extra_message is not None:
+          message += '. ' + extra_message
       message += '. Use %s instead.'% current if current else '.'
```

Comment:
Two comments:
- There is a missing space at `.'%`
- `extra_message` should probably come after all other default messages.